cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/BrokerFailureDetector.java

```diff
@@ -44,8 +44,12 @@
 public class BrokerFailureDetector {
   private static final Logger LOG = LoggerFactory.getLogger(BrokerFailureDetector.class);
   public static final String FAILED_BROKERS_OBJECT_CONFIG = "failed.brokers.object";
+  public static final String BROKER_FAILURES_FIXABLE_CONFIG = "broker.failures.fixable.object";
   private static final String ZK_BROKER_FAILURE_METRIC_GROUP = "CruiseControlAnomaly";
   private static final String ZK_BROKER_FAILURE_METRIC_TYPE = "BrokerFailure";
+  // TODO: make the threshold configurable
+  private static final int FIXABLE_BROKER_FAILURE_UPPER_BOUNDARY = 5;
+  private static final double FIXABLE_BROKER_FAILURE_PERCENTAGE_UPPER_BOUNDARY = 0.2;
```

Comment:
`0.2` means that in a cluster of `9` brokers, more than 1 broker failure cannot be handled. This seems a little aggressive to me.

Can we set the default `FIXABLE_BROKER_FAILURE_PERCENTAGE_UPPER_BOUNDARY ` and `FIXABLE_BROKER_FAILURE_UPPER_BOUNDARY` to less aggressive values to avoid a dramatic change in the existing behavior?